ui/Main.elm

```diff
@@ -59,35 +59,41 @@ update msg ({ filterValues } as model) =
 
         UpdateFilter newFilter ->
             let
+                noFilters =
+                    { model
+                        | productFilter = "all"
+                        , versionFilter = "all"
+                        , platformFilter = "all"
+                        , channelFilter = "all"
+                        , localeFilter = "all"
+                        , buildIdFilter = ""
```

Comment:
Indeed! I had submitted #65 some time ago, but didn't get to it yet, as it seems less urgent